x-pack/plugins/timelines/server/search_strategy/timeline/factory/events/details/query.events_details.dsl.ts

```diff
@@ -5,25 +5,43 @@
  * 2.0.
  */
 
+import { JsonObject } from '@kbn/common-utils';
 import { DocValueFields } from '../../../../../../common/search_strategy';
 
 export const buildTimelineDetailsQuery = (
   indexName: string,
   id: string,
-  docValueFields: DocValueFields[]
-) => ({
-  allowNoIndices: true,
-  index: indexName,
-  ignoreUnavailable: true,
-  body: {
-    docvalue_fields: docValueFields,
-    query: {
-      terms: {
-        _id: [id],
-      },
+  docValueFields: DocValueFields[],
+  authFilter?: JsonObject
+) => {
+  const basicFilter = {
+    terms: {
+      _id: [id],
     },
-    fields: [{ field: '*', include_unmapped: true }],
-    _source: true,
-  },
-  size: 1,
-});
+  };
+  const query =
+    authFilter != null
+      ? {
+          bool: {
+            filter: [{ ...basicFilter }, { ...authFilter }],
```

Comment:
do we really need the spreading here
```suggestion
            filter: [basicFilter, authFilter],
```

Reply:
Guess we don't need to worry about mutations to authFilter here so no. I think I'm in the habit of using the spread when returning or assigning values to not copy by reference, but fine removing that here.